regulations/static/regulations/js/source/views/comment/comment-view.js

```diff
@@ -44,6 +36,7 @@ var CommentView = Backbone.View.extend({
   initialize: function(options) {
     this.$context = this.$el.find('.comment-context');
     this.$container = this.$el.find('.editor-container');
+    this.$input = this.$el.find('input[type="file"]');
     this.$queued = this.$el.find('.queued');
```

Comment:
Might be worth renaming this now, given that some of the uploads are "queued" in a different sense.